build.sh

```diff
@@ -12,11 +12,7 @@ cmake -DCMAKE_BUILD_TYPE:String=$BUILD_TYPE \
       -DUSE_PSPLINE=ON \
       -DUSE_FIO=OFF \
       -DKORC_TEST=OFF \
-      -DCMAKE_Fortran_FLAGS="-DHDF5_DOUBLE_PRESICION -malign-double -fconvert='big-endian'" \
-      -DCMAKE_C_FLAGS="-malign-double"  \
-      -DCMAKE_CXX_FLAGS="-malign-double" \
-      -DCMAKE_Fortran_FLAGS_DEBUG="-g3 -ffpe-trap=zero,overflow -fbacktrace" \
-      -DCMAKE_C_FLAGS_DEBUG="-g3" \
-      -DCMAKE_CXX_FLAGS_DEBUG="-g3" ../
+      -DCMAKE_Fortran_FLAGS="-DHDF5_DOUBLE_PRESICION" \
+    ../
```

Comment:
We want either separate build.sh files for `gfortran` and `nvfortran` or different FLAGS based on a input parameter?